spec/helpers/serializer_helper_spec.rb

```diff
@@ -2,13 +2,12 @@
 
 require 'spec_helper'
 
+class ExampleEnterpriseSerializer < ActiveModel::Serializer
+  attributes :id, :name
+end
+
 describe SerializerHelper, type: :helper do
-  let(:serializer) do
-    class ExampleEnterpriseSerializer < ActiveModel::Serializer
-      attributes :id, :name
-    end
-    ExampleEnterpriseSerializer
-  end
+  let(:serializer) { ExampleEnterpriseSerializer }
```

Comment:
The class definition is moved here but we still have the problem that we define a new class in a spec file which will be visible to all other specs running after it, but not to specs before it. It's leaking state. I wonder if we can create an anonymous class like in this spec:
https://github.com/openfoodfoundation/openfoodnetwork/blob/eac278ea1f591d04a99df1e43f2f66be0a2d5d62/spec/lib/tasks/reset_spec.rb#L13-L15